airlock/nav.py

```diff
@@ -33,9 +35,13 @@ def iter_nav(items, request):
 def menu(request):
     items = [
         NavItem(name="Workspaces", url_name="workspace_index"),
-        NavItem(name="Requests", url_name="request_index"),
+        NavItem(name="Requests", url_name="requests_for_researcher"),
         NavItem(name="Docs", url_name="docs_home"),
     ]
+
+    if request.user and permissions.user_can_review(request.user):
+        reviews_menu = NavItem(name="Reviews", url_name="requests_for_output_checker")
```

Comment:
I'm not 100% sold on "Reviews" being the menu item name, because it's slightly overloaded with the use of "reviews" elsewhere - but I can't come up with anything better...! "Requests for review" is (for me) clearer but too wordy to be a menu button! hmmmm